content/reference/sql/version-control/dolt-system-tables.md

```diff
@@ -756,6 +757,141 @@ WHERE  to_commit='pu60cdppae7rumf1lm06j5ngkijp7i8f';
+
+### Example Query
+
+Taking the
+[`dolthub/nba-players`](https://www.dolthub.com/repositories/dolthub/nba-players)
+database from [DoltHub](https://www.dolthub.com/) as our
+example, the following query uses the `dolt_column_diff` system table to find all commits, and tables where the game played stat was updated.
+
+```sql
+SELECT commit_hash, table_name
+FROM   dolt_column_diff
+WHERE  column_name = 'gp';
+```
+
+```text
++----------------------------------+------------------------------+-------------+
+| commit_hash                      | table_name                   | column_name |
++----------------------------------+------------------------------+-------------+
+| jm0d2j8doi3mj1n4nu1lr530r5fqle5r | career_totals_allstar        | gp          |
+| jm0d2j8doi3mj1n4nu1lr530r5fqle5r | career_totals_post_season    | gp          |
+| jm0d2j8doi3mj1n4nu1lr530r5fqle5r | career_totals_regular_season | gp          |
+| jm0d2j8doi3mj1n4nu1lr530r5fqle5r | season_totals_allstar        | gp          |
+| jm0d2j8doi3mj1n4nu1lr530r5fqle5r | season_totals_post_season    | gp          |
+| jm0d2j8doi3mj1n4nu1lr530r5fqle5r | season_totals_regular_season | gp          |
+| 41il57n6opimpubmacng9cvh1hmi92t5 | career_totals_allstar        | gp          |
+| 41il57n6opimpubmacng9cvh1hmi92t5 | career_totals_post_season    | gp          |
+| 41il57n6opimpubmacng9cvh1hmi92t5 | career_totals_regular_season | gp          |
+| 41il57n6opimpubmacng9cvh1hmi92t5 | season_totals_allstar        | gp          |
+| 41il57n6opimpubmacng9cvh1hmi92t5 | season_totals_post_season    | gp          |
+| 41il57n6opimpubmacng9cvh1hmi92t5 | season_totals_regular_season | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | career_totals_allstar        | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | career_totals_post_season    | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | career_totals_regular_season | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | rankings_post_season         | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | rankings_regular_season      | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_allstar        | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_post_season    | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_regular_season | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | career_totals_allstar        | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | career_totals_post_season    | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | career_totals_regular_season | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | rankings_post_season         | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | rankings_regular_season      | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_allstar        | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_post_season    | gp          |
```

Comment:
I would suggest omitting some of this row data; maybe use an ellipsis after showing 5 rows to indicate that there is more content, and then show 2 or 3 rows at the end. Otherwise, I think this just has too much "visual weight" in the docs that doesn't add to helping the reader understand how the system table works – I think they could get the same understand from seeing a couple of sample rows.